sycl/include/sycl/detail/pi.h

```diff
@@ -1835,6 +1836,16 @@ __SYCL_EXPORT pi_result piTearDown(void *PluginParameter);
 /// runtime must handle it or end the application.
 __SYCL_EXPORT pi_result piPluginGetLastError(char **message);
 
+/// Returns the global timestamp from \param device , and syncronized host
+/// timestamp
+///
+/// \param device device to query for timestamp
+/// \param deviceTime pointer to store device time
+/// \param hostTime pointer to store syncronized host time
```

Comment:
I think it makes sense to make both timestamps optional (allow nullptr)

Reply:
Should it be required to at least one pointer not be mull?

Reply:
It certainly makes sense for user to request at least one, but I don't think we should require this.